cli/cmd/sync_v0_2.go

```diff
@@ -114,7 +115,7 @@ func syncConnectionV0_2(ctx context.Context, cqDir string, sourceClient *source.
 			if err := destClients[i].Close(ctx); err != nil {
 				return fmt.Errorf("failed to close destination client for %s -> %s: %w", sourceSpec.VersionString(), destination.VersionString(), err)
 			}
-			failedWrites += destFailedWrites
+			atomic.AddUint64(&failedWrites, destFailedWrites)
```

Comment:
So this test detected a data race 🙃  We can release as `fix` but seems not that important